CosmosToRedis/CosmosToRediscs.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using StackExchange.Redis;
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+using Container = Microsoft.Azure.Cosmos.Container;
+
+namespace CosmosToRedis
```

Comment:
Create a consistent namespace "Microsoft.Azure.WebJobs.Extensions.Redis.Samples" for example.

Reply:
Please review among the three of you how you would like to structure all of your code.